### PR TITLE
fixing links to files for metadata

### PR DESCRIPTION
**Pull request by boyney123**
Fix for a few files that have broken links to files.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json [hunk 1]
```diff
@@ -3,22 +3,18 @@
     "description": "This project contains unit and integration tests for a pattern using API Gateway, AWS Lambda and Amazon DynamoDB.",
     "content_language": "English",
     "language": ".NET",
-    "type": ["Unit", "Integration"],
+    "type": ["Integration"],
```
[the remark this answers is not in the document]
Comment (boyney123) [reply dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json:6]: From what I understand the pattern is only showing an integration test?

```
 "pattern_detail_tabs": [
        {
            "title": "Application Code",
            "filepath": "/src/ServerlessTestSamples/Function.cs"
        },
        {
            "title": "Integration Test",
            "filepath": "/tests/ServerlessTestSamples.IntegrationTest/IntegrationTest.cs"
        }        
    ],
 ```
 
 Others have unit + integration in the array, so I thought I would remove the tag for Unit, as I can't see one?

[the remark this answers is not in the document]
Comment (dhelper) [reply dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json:6]: We have both, I guess I accidently did not add the unit test folder (test/ServerlessTestSamples.UnitTest

**Comment by brnkrygs** [comment]
TypeScript changes look good. Though, I'm a bit thrown because from what I can see, `main` already has fixed links for that sample:

https://github.com/aws-samples/serverless-test-samples/blob/main/typescript-test-samples/apigw-lambda-dynamodb/metadata.json

File: dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (wtromano-aws) [reply dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json:6]: Thank you, good find here, let's move forward with just "integration" for this MR, and we can follow up with a separate MR to add the unit tests to this patter.

File: dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json [hunk 2]
```diff
@@ -3,22 +3,18 @@
...
     "services": ["apigw", "lambda", "s3"],
     "pattern_detail_tabs": [
         {
             "title": "Application Code",
-            "filepath": "/src/"
-        },
-        {
-            "title": "Unit Tests",
-            "filepath": "/tests/ServerlessTestSamples.UnitTest/ApiTests.UnitTest"
+            "filepath": "/src/ServerlessTestSamples/Function.cs"
```
[the remark this answers is not in the document]
Comment (wtromano-aws) [reply dotnet-test-samples/apigw-lambda-list-s3-buckets/metadata.json:13]: Should have multiple - array of {title, filepath}  to the key Application code and test code.

File: python-test-samples/lambda-mock/metadata.json
Comment (wtromano-aws) [inline python-test-samples/lambda-mock/metadata.json:1]: This change is already in the main branch

**Review by wtromano-aws — CHANGES_REQUESTED** [review]
I think some of the linking issues may already be resolved in a recent merge, as I am seeing redundant changes.  Perhaps fast-forward to the changes already in main?

**Comment by brnkrygs** [comment]
Closing as abandoned. Repo structure has evolved since this was opened.

**Pull request closed without merging**